db/migrate/20210311225231_create_jwt_denylist.rb

```diff
@@ -0,0 +1,9 @@
+class CreateJwtDenylist < ActiveRecord::Migration[6.0]
+  def change
+    create_table :jwt_denylist do |t|
```

Comment:
Não daria pra criar essa tabela nomeada no padrão que o rails pede? Pra não ter que especificar a table_name la no model?

Reply:
Daria, mas geralmente quando se usa o devise acho que por convenção a gente segue as [docs do devise_jwt](https://github.com/waiting-for-dev/devise-jwt#denylist).